pos-print/src/main/java/com/clouway/pos/print/adapter/http/HttpModule.java

```diff
@@ -22,6 +22,7 @@ protected void configureSitebricks() {
         at("/_status").serve(StatusService.class);
         at("/v1/posprint").serve(PrintService.class);
         at("/v1/devices").serve(CashRegistersService.class);
+        at("/v1/report").serve(ReportService.class);
```

Comment:
I'm opening discussion for naming here. 

About class structure, maybe some duplication between both services can be removed if one service is used instead, but it need to be considered.